### ISSUE TITLE
Wizard steps can be skipped

### ISSUE THREAD

**Issue by austincarpenter** [filamentphp/filament#2998]
### Package

filament/forms

### Package Version

v2.13.14

### Laravel Version

v9.18

### Livewire Version

v2.10.6

### PHP Version

PHP 8.16

### Bug description

Clicking rapidly on the Next button in a Wizard allows for steps skip validation. This probably isn't an issue if explicitly validating the entire form on the submit action but could have some unforeseen consequences for various use cases. Haven't looked into implications inside the Admin Panel just yet.

https://user-images.githubusercontent.com/3736774/176814543-a7be59e6-6a67-4b04-9967-5225a0b9ff49.mov

### Steps to reproduce

```php
protected function getFormSchema(): array
{
    return [
        Wizard::make([
            Wizard\Step::make('Step 1')
                ->schema([
                    TextInput::make('input-1')
                        ->required(),
                ]),
            Wizard\Step::make('Step 2')
                ->schema([
                    TextInput::make('input-2')
                        ->required(),
                ]),
            Wizard\Step::make('Step 3')
                ->schema([
                    TextInput::make('input-3')
                        ->required(),
                ]),
        ]),
    ];
}
```

**Comment by simonbuehler** [comment]
could you try to remove `delay` from https://github.com/filamentphp/filament/blob/472747b628a13265aa5728ed0fd60983097ddcb9/packages/forms/resources/views/components/wizard.blade.php#L212

**Comment by ryangjchandler** [comment]
Yeah, seems like an appropriate fix to remove the `delay` modifier here.

**Comment by austincarpenter** [comment]
Apologies, been quite busy to get back around to this one! Thanks for the fix.
